muesli/web/templates/lecture/view_points.pt

```diff
@@ -24,8 +24,11 @@
       </tr>
     <tr>
       <th>Insgesamt</th>
-      <td colspan="2"><strong>${fF(catexams['sum'])}/${fF(catexams['max'])}</strong>
-        <span tal:condition="catexams['max']>0">
+      <td colspan="3"><strong>${fF(catexams['sum'])}/${fF(catexams['max'])}</strong>
+        <span tal:condition="catexams['max']>0 and catexams['max']!=catexams['max_rel']">
+         (${'%2.f' % (catexams['sum']/catexams['max']*100)}%, ${'%2.f' % (catexams['sum']/catexams['max_rel']*100)}% rel.)
```

Comment:
Division by zero if `catexams['max_rel']` is 0 ... 🤔

Reply:
Yep, you're absolutely right, I completely overlooked that :D